spec/datarockets/style/cop/layout/array_alignment_extended_spec.rb

```diff
@@ -0,0 +1,336 @@
+# frozen_string_literal: true
+
+# rubocop:disable RSpec/ExampleLength
```

Comment:
As you see I've disabled this rule for specs in this case because I want to check autocorrector too. But if we have `expect_offense` and `expect_correction` in one example and we test **multi-line** arrays, 10 line are not enough 😬

Reply:
You can use arrays with less number of literal in test cases, 3 instead of 4. You can try, maybe there is a chance to fix the issue

Reply:
I've thought about it but it is still not enough even for 3 literal tests, and to be honest 4 literals is more visual for some tests